SPECS/nvidia-container-runtime/nvidia-container-runtime.spec

```diff
@@ -0,0 +1,60 @@
+Summary:        NVIDIA container runtime
+Name:           nvidia-container-runtime
+Version:        3.4.2
+Release:        2%{?dist}
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/NVIDIA/nvidia-container-runtime
+#Source0:       https://github.com/NVIDIA/%%{name}/archive/v%%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+BuildRequires:  golang
+Requires:       libseccomp
+
+%description
+Provides a modified version of runc allowing users to run GPU enabled
+containers.
+
+%prep
+%setup -q
+
+%build
+cd src/
+make build
+mkdir -p %{buildroot}%{_bindir}
+cp nvidia-container-runtime %{buildroot}%{_bindir}
+
+%install
+cd src
+install -m 755 nvidia-container-runtime %{buildroot}%{_bindir}/%{name}
```

Comment:
Could we consistently use either `nvidia-container-runtime` or `%{name}`?

Reply:
changed to use the macro %{name}